src/boot/bootctl.c

```diff
@@ -1139,13 +1141,13 @@ static int install_variables(const char *esp_path,
                                        "Failed to determine current boot order: %m");
 
         if (first || r == 0) {
-                r = efi_add_boot_option(slot, "Linux Boot Manager",
+                r = efi_add_boot_option(slot, arg_efi_label ?: "Linux Boot Manager",
```

Comment:
let's not have this ternary op at multiple places. Maybe add a short helper that hides it internally:

```c
static const char *pick_efi_boot_option_description(void) {
        return arg_efi_boot_option_description ?: "Linux Boot Manager";
}
```

And then call this function whenever needed.